pkgs/dart_services/lib/src/flutter_genui.dart

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2014, the Dart project authors.  Please see the AUTHORS file
```

Comment:
nit: for the copyright header you want to use the year the file was created:
```suggestion
// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
```